src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo"
+        src="images/logo.png"
+        alt="#">
+      </a>
+      <nav>
+        <ul class="nav">
+          <a class="nav_link is-active" href="#">Apple</a>
+          <a class="nav_link" href="#">Samsung</a>
+          <a class="nav_link" href="#">Smartphones</a>
+          <a class="nav_link"href="#"
```

Comment:
Missing space between attributes 'class' and 'href'.

Reply:
![man-mechanic_1f468-200d-1f527](https://github.com/mate-academy/layout_moyo-header/assets/129754902/8c5fde0c-21f7-421e-a92b-9e9649b8a53b)

Reply:
the same